### PR TITLE
feat(validation): css & yup validation

### PR DESCRIPTION
**Pull request by kibo-amolpawar**

### CODE REVIEW

File: assets/styles/components/molecules/SfSelect.scss [hunk 1]
```diff
@@ -24,6 +24,23 @@
     border-radius: var(--spacer-2xs);
     height: calc(var(--spacer-lg) + 0.125rem);
     border: 1px solid var(--_c-gray-middle);
-    margin-top: 15px;
+  }
+
+  &.country {
+    @include for-desktop {
+      height: 36px;
+      bottom: 20px;
```
Comment (kibo-susantasamanta) [inline assets/styles/components/molecules/SfSelect.scss]: Should use variables instead of hard coding it.

File: pages/checkout.vue [hunk 2]
```diff
@@ -521,17 +517,18 @@ export default {
         case stepLabels.GO_TO_SHIPPING: {
           if (!isValidPersonalDetails.value) return
           await savePersonalDetails()
+          enableNextStep.value = false
```
Comment (kibo-kevinwatts) [inline pages/checkout.vue:520]: why are we setting to false here?

Comment (kibo-amolpawar) [reply pages/checkout.vue:520]: When we are redirecting to the next step from the valid step. The checkout button should be disabled until that step is not valid so it's added here.

File: assets/styles/components/molecules/SfSelect.scss (hunk 1 above)
Comment (kibo-amolpawar) [reply assets/styles/components/molecules/SfSelect.scss]: Resolved

**Review by kibo-kevinwatts — APPROVED** [review]

**Pull request merged**